sos/cleaner/mappings/mac_map.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Red Hat, Inc. Jake Hunsaker <jhunsake@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import random
+import re
+
+from sos.cleaner.mappings import SoSMap
+
+
+class SoSMacMap(SoSMap):
+    """Mapping store for MAC addresses
+
+    MAC addresses added to this map will be broken into two halves, vendor and
+    device like how MAC addresses are normally crafted. For the vendor hextets,
+    obfuscation will take the form of 53:4f:53, or 'SOS' in hex. The following
```

Comment:
>53:4f:53, or 'SOS'
That's a nice touch :)

I was hoping to come up with a proposal to not filter the first 3, but the list (https://gist.github.com/aallan/b4bb86db86079509e6159810ae9bd3e4) is really long and defense orgs have claimed some of them... so oh well.

Reply:
I think we may want to not filter 01:00:5e* or 33:33:00*  which from a quick search appear to be multicast macs.